documentation/making_a_config.md

```diff
@@ -51,36 +51,62 @@ input_options:
 ```
 
 ## Bulk Filters
-This is where you specify how you would like to downselect the material design space. There are several options:
-1. Bulk ids to include: a list of bulk ids to include
+This is where you specify how you would like to downselect the material design space. There are several options which are listed here and detailed below:
+
+1. `filter_by_bulk_ids`
+2. `filter_by_acceptable_elements`
+3. `filter_ignore_bulk_ids`
+4. `filter_by_num_elements`
+5. `filter_by_required_elements`
+6. `filter_by_object_size`
+7. `filter_by_elements_active_host`
+8. `filter_by_element_groups`
+9. `filter_by_bulk_e_above_hull`
+10. `filter_by_bulk_band_gap`
+11. `filter_by_pourbaix_stability`
+12. `filter_fraction`
+
+
+### 1. Bulk ids to include: 
+a list of bulk ids to include
 ```filter_by_bulk_ids: ['mp-126','mp-30', 'mp-81', 'mp-13', 'mp-79']```
-2. Acceptable elements: a list of element symbols to include. If you say "Au" and "Ag", and materials containing only Au and Ag will be included.
+### 2. Acceptable elements:
+a list of element symbols to include. If you say "Au" and "Ag", and materials containing only Au and Ag will be included.
 ```filter_by_acceptable_elements: ["Au", "Ag"]```
-3. Bulk ids to ignore: a list of bulk ids to exclude
+### 3. Bulk ids to ignore:
+a list of bulk ids to exclude
 ```filter_ignore_bulk_ids: ['mp-126','mp-30', 'mp-81', 'mp-13', 'mp-79']```
-4. Number of elements: a list of the numbers which are acceptable i.e. `[2, 3]` gives you binary and ternary materials
+### 4. Number of elements:
+a list of the numbers which are acceptable i.e. `[2, 3]` gives you binary and ternary materials
 ```filter_by_num_elements: [2, 3]```
-5. Required elements: a list of elements which must be in each material. If you say "Cu", then all copper alloys will be considered.
+### 5. Required elements:
+a list of elements which must be in each material. If you say "Cu", then all copper alloys will be considered.
 ```filter_by_required_elements: ["Cu"]```
-6. Number of atoms: this is useful to avoid very large structures which are costly to compute. It filters out all unit cell structures with more atoms than the number specified.
+### 6. Number of atoms:
+this is useful to avoid very large structures which are costly to compute. It filters out all unit cell structures with more atoms than the number specified.
```

Comment:
Might mention that people can use the Sankey diagram to calibrate this value? Or maybe people running this will have some intuition

Reply:
yeah I think since the sankey diagram is constructed at the end of a run its not actually very useful for this purpose. We could provide a suggested value to get to this point though?